ECOSYSTEM.md

```diff
@@ -8,6 +8,7 @@ Plugins and tools maintained by the knex team are listed under [Core](#core) whi
 - [`knex-tablecleaner`](https://github.com/kibertoad/knex-tablecleaner) Library for deleting all rows from a given list of DB tables.
 
 #### [Community](#community)
+- [`knemm`](https://github.com/asteinarson/knemm) A CLI tool to manage SQL schema in declarative YAML files (`claims` and `states`). It allows for database dependencies between loosely coupled modules. 
```

Comment:
It feels like a verb is missing there. `It allows for database dependencies between loosely coupled modules. `
Should it perhapse be `for handling` or `for managing dependencies between loosely coupled modules`?

Reply:
That's a bit too involved for me - feel free to insert those words, if you distinctly feel it improves the description.  (For me the important thing is to roughly communicate an idea - and only by looking directly on it will they get a more full picture). 

I think peoples background and expectations w regards to spoken/written english varies quite a bit, even what is considered correct - from place to place, from person to person. 

Strictly speaking I really don't think anything is wrong in the sentence. But feel free to modify it, if you feel confident it is better that way. 

The main thing (for me) is to communicate approximately what it is about, the name of the repo and the link. 

Thanks.